bakerydemo/static/css/main.css

```diff
@@ -427,6 +445,11 @@ li.has-submenu a.allow-toggle {
 .breadcrumb-container + content > .hero {
   margin-top: -36px;
 }
+@media (max-width: 768px) {
+  .breadcrumb-container + content > .hero {
+    margin-top: 0px;
+  }
```

Comment:
Some issues:

- There is no `content` tag in HTML.
- From what I can see this is fixing the `margin-top: -36px` breaking the layout on mobile.
- (0 should be unitless)

This would be easier to achieve by changing the `.breadcrumb-container` definition above to be `position: absolute;` and `left: 0; right: 0;`, to produce the same effect without having to change the styles of multiple elements. Then the two `.breadcrumb-container + content > .hero` can be removed.